### PR TITLE
crystalline 0.10.0 (new formula)

### PR DESCRIPTION
**Pull request by y8**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x]  Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x]  Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x]  Have you built your formula locally with brew install --build-from-source <formula>, where <formula> is the name of the formula you're submitting?
- [x]  Is your test running fine brew test <formula>, where <formula> is the name of the formula you're submitting?
- [x]  Does your build pass brew audit --strict <formula> (after doing brew install --build-from-source <formula>)? If this is a new formula, does it pass brew audit --new <formula>?

### CODE REVIEW

File: Formula/crystalline.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  desc "Language Server Protocol implementation for Crystal"
+  homepage "https://github.com/elbywan/crystalline"
+  url "https://github.com/elbywan/crystalline/archive/refs/tags/v0.10.0.tar.gz"
+  sha256 "26c926ba423e4b04fc52af501cd842c8255312014fc4aa1bc3735a8cd0df3426"
+  license "MIT"
+
+  livecheck do
+    url :homepage
+  end
+
```
Comment (chenrui333) [inline Formula/crystalline.rb:11]:
```suggestion
```

Comment (y8) [reply Formula/crystalline.rb:11]: Thanks for suggestion! Just to be sure: livecheck is not required anymore? Can't find more info about modern best practices around it.

Comment (p-linnane) [reply Formula/crystalline.rb:11]: Livecheck will run using the `:git` strategy on all GitHub URL's by default. No need to specify one unless the default isn't working.

**Review by chenrui333 — APPROVED** [review]
LGTM

**Comment by chenrui333** [comment]
@y8, can you squash all the commits into one? Thanks!

**Comment by y8** [comment]
> @y8, can you squash all the commits into one? Thanks!

Done!

**Review by chenrui333 — APPROVED** [review]

**Review by p-linnane — APPROVED** [review]

**Pull request merged**